### PR TITLE
Task 8

### PR DESCRIPTION
**Pull request by Ksenia000**

### CODE REVIEW

File: Learn/Test.html [hunk 1]
```diff
@@ -1,20 +1,53 @@
 <html>
   <head>
   <link href = "style.css" rel = "stylesheet">
+  <link href = "css/style_grid.css" rel = "stylesheet">
+    <link href = "css/my_style.css" rel = "stylesheet">
```
Comment (akryvenko) [inline Learn/Test.html]: Correct style. Add more space before **link** tag

File: Learn/Test.html [hunk 2]
```diff
@@ -1,20 +1,53 @@
...
+			<p class = "div-class div-2">Result (dd-mm-yyyy):</p>
+			<p id = "div-4" class = "div-class">Result (dd/mm/yyyy):</p>
+			<!--<p id = "test" >Result :</p>-->
+			<button OnClick = "CalcToDiv()" class = "btn btn-2">Run</button> 
+		</div>
+		</fieldset>
+		</div>
+		<div class=" cols col-6 colTask" >
+		  <fieldset>
+	      <legend>Num Game</legend>
```
Comment (akryvenko) [inline Learn/Test.html:28]: Please correct line indent

File: Learn/js/num_script.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+	/*else{
+		if (num == randInt){
+		document. getElementById( "res" ).innerHTML ="Good Work!";
+		return false;}
+		else{
+		document. getElementById( "res" ).innerHTML ="Not matched!";
+		return false;	
+		}
+	}*/
+	// return false;
```
Comment (akryvenko) [inline Learn/js/num_script.js]: Please remove these comments. It is redundant

File: Learn/js/num_script.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
+ function validate (){ 
+ 	
+//Считаем значения из поля 
+	var num = document.getElementById("field"). value ;
+	var randInt = Math.floor(Math.random() * (10 - 1 + 1) + 1);
+	if (isNaN(num) || num.length == 0 || num < 1 && num > 0) {
```
Comment (akryvenko) [inline Learn/js/num_script.js]: no validation for scenario when num > 10

File: Learn/js/num_script.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+		document.getElementById( "resErr" ).innerHTML ="*INCORRECT! Enter number between 1 to 10, please";
+	} else {
+		if (num == randInt){
+			document. getElementById( "res" ).innerHTML ="Good Work!";
+		} else {
+			document. getElementById( "res" ).innerHTML ="Not matched!Try again=)";
```
Comment (akryvenko) [inline Learn/js/num_script.js]: Please add more spaces to message. e..g "Not matched! Try again =)"

**Review by akryvenko — CHANGES_REQUESTED** [review]

File: Learn/Test.html (hunk 1 above)
Comment (Ksenia000) [reply Learn/Test.html]: done

File: Learn/Test.html (hunk 2 above)
Comment (Ksenia000) [reply Learn/Test.html:28]: done

File: Learn/js/num_script.js (hunk 4 above)
Comment (Ksenia000) [reply Learn/js/num_script.js]: опечатка!!!!!!!!!!

File: Learn/js/num_script.js (hunk 5 above)
Comment (Ksenia000) [reply Learn/js/num_script.js]: booo

**Review by akryvenko — APPROVED** [review]

**Pull request merged**